Client/src/index.css

```diff
@@ -11,6 +11,27 @@
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
+  --size-40: 40px;
+  --size-38: 38px;
+  --size-35: 35px;
+  --size-28: 28px;
+  --size-24: 24px;
+  --size-20: 20px;
+  --size-16: 16px;
+  --size-13: 13px;
```

Comment:
Overall I think this is generally what we had in mind, but if we name the vars in a way that reflects their values we kind of defeat the purpose refacotring the values.

The idea behind refactoring out the values is to give us the flexibility to change them all in one place if we want to later.  If for example we want to change the `13px` value to `16px` the var will now be

`--size-13: 16px`

which works, but semantically no longer makes sense.  I think it would be best to name these something like `--size-small` `--size-extra-small` etc.

I'll leave it up to the front end guys to chime in on this @Skorpios604  @MuhammadKhalilzadeh

Reply:
This is just a temporary naming. My main goal is to circularly give my stylings general names (Like what you mentioned) and add my styles to the Theme after completing the design of every couple of pages.
I just first want to see how frequently each value gets repeated and what's the range it can have and what name could best describe it.

Reply:
@MuhammadKhalilzadeh I think these vars were created by @alvarotorrestx ?

In any event I'll leave this one up to you guys to decide on :+1:

Reply:
I agree on the names being more general like @popnfresh234 stated with --size-small, etc. Although, I had created them with multiple names since there were many different, yet repeated, styles. I'm sure like @MuhammadKhalilzadeh said, once there's consistent styles this will come with ease to rename and readjust the values.

Reply:
@alvarotorrestx I'm OK with that :+1: Once our team lead @veyselboybay has a chance to review I think we can probably merge this in

Reply:
I think the overall idea of the naming convention that @alvarotorrestx came up with is good practice because of reusability, maintainability, and it will also be good for our theming. I also like the readability of it. I also think it is important to consider what @popnfresh234 said with regard to the semantics of the naming convention.